WebRoot/css/zm.css

```diff
@@ -7279,4 +7269,78 @@ input[type="radio"] + label {
 
 .ZmWhiteBlackViewNumberUsed {
 	@WhiteBlackListViewItemsNumberUsed@
-}
\ No newline at end of file
+}
+
+/* Preference signature interface panel */
+.ZmPrefSignaturePage {
+	@PrefSignaturePage@
+}
+
+.ZmPrefSignaturePage hr {
+	@PrefSignaturePageSeparator@
+}
+
+.ZmSignatureNameWrapper {
+	@PrefSignatureNameWrapper@
+}
+
+.ZmSignatureNameWrapper > label {
+	@PrefSignatureNameLabel@
+}
+
+.ZmSignatureNameWrapper .DwtInputField input[type='text'] {
+	@PrefSignatureNameInput@
+}
+
+.ZmSignatureFormatWrapper {
+	@PrefSignatureFormatWrapper@
+}
+
+.ZmSignatureListWrapper {
+	@PrefSignatureListWrapper@
+}
+
+.ZmSignatureListView {
+	@PrefSignatureListView@
+}
+
+.ZmSignatureListView [class*='Row'] TD {
```

Comment:
Yes, Row class would always applies. But based on state, sometimes along with `.Row` other class are also get applied. For instance `Row-Selected` or `Row-Focussed` and in any of these cases, I want this rules to get applied. (I am not sure though if any-of those class will break UI or not. Just that I want to be on safer side :) )

Other way to handle this would be to have my selectors as follows:

```
.ZmSignatureListView .Row TD,
.ZmSignatureListView .Row-focussed TD,
.ZmSignatureListView .Row-selected TD,
.ZmSignatureListView .Row-disabled TD {
   //my rules
}
```

Here I am just avoiding this repetition with this selector. 

Thanks